app/features/main.py

```diff
@@ -17,7 +17,7 @@ def main(argv):
     print(mid.tempos.getAll())
 
     print('= AVERAGE TEMPOS TIME =')
-    print(mid.tempos.getAverageByPartsTime())
+    print(mid.tempos.getAverageByPartsTime(1))
```

Comment:
What happens with nbSplit == 3 when we have only one tempo msg ?

Reply:
there is just one value that is displayed (the tempo value)